submissions/NwagboJohn/my_php/show.php

```diff
@@ -0,0 +1,44 @@
+
+<!DOCTYPE html>
+  <html>
+    <head>
+      <link rel="stylesheet" type="text/css" href="index.css">
+    </head>
+  ` <body>
+  
+      <h1>AVAILABLE STOCK</h1>
+
+      <?php
+         $servername = "localhost";
+          $username = "root";
+          $password = "";
+          $dbname = "store";
+
+          // Create connection
+          $conn = new mysqli($servername, $username, $password, $dbname);
+          // Check connection
+          if ($conn->connect_error) {
+             die("Connection failed: " . $conn->connect_error);
+          }
+
+          $sql = "SELECT id, brand, model,price,quantity,dates FROM dataEntry ";
+          $result = $conn->query($sql);
+
+          if ($result->num_rows > 0) {
+              echo  "<table><tr><th>SERIAL NO</th><th>BRAND</th><th>MODEL</th><th>PRICE</th><th>QUANTITY</th><th>DATE</th></tr>";

```

Comment:
Try to break down this table for readability. You can write it as you would write the markup for a table with HTML. 
<table> on it's own line, <tr> on it's own line, and so on